CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+# Copyright 2022
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php.
```

Comment:
Should probably use the same copyright header as every other file in the project. i.e:
```bash
/***********************************************************************
 * Copyright (c) 2022                                                                                      *
 * Distributed under the MIT software license, see the accompanying      *
 * file COPYING or https://www.opensource.org/licenses/mit-license.php. *
   ***********************************************************************/
```

Reply:
CMake language have no `/*...*/` [comments](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#comments).